### DISCUSSION TITLE
2022/05/我拿Emacs做什么暨Emacs随记（2）/

### DISCUSSION THREAD

**Discussion by SouthFox-D** [SouthFox-D/SouthFox-D.github.io#70] [博客评论]
https://blog.southfox.me/2022/05/我拿Emacs做什么暨Emacs随记（2）/

**Comment by windnore** [comment]
您好，“蔡格尼克效应”的发现者 Bluma Zeigarnik 是一位女性（参见 https://en.wikipedia.org/wiki/Bluma_Zeigarnik），方便的话可以请您将相关的人称代词修改为“她”吗？
如果您本身不习惯使用“她”这个代词，也可以无视这条评论。我只是想提一下 Bluma 的女性身份，如果您先前不知道/没有意识到这点的话。
如有打扰或冒犯，我对此感到很抱歉。

**Reply by SouthFox-D** [reply]
已改进，参见 https://github.com/SouthFox-D/SouthFox-D.github.io/commit/dff7c055e3eeae3c2f8c6184129ab03eed0c9c9c
